fw/config.c

```diff
@@ -0,0 +1,36 @@
+#define CONFIG_VERSION 0
```

Comment:
What purpose does this serve?

Reply:
Since EEPROM is initialised to `0xFF`, we need a way to know which config values to set if a new field gets added. So a config versioning system allows us to reset certain fields instead of everything.

Reply:
I guess we could just check which fields are set to `0xFF`, but I think a versioning system is more robust.

Reply:
Agreed.